The last commit 39cbf24b2b4dd0557a0e0f07cc835a2588424805 backs out some unit removal that was required earlier in this branch when we had fewer dependencies.  This implies that the dependencies installed (more now) changes the test results. 

😨 

Would be nice to understand this.  Perhaps looking for

```python
try:
    import foo
except:
    pass
```

that silently changes tables and units that are read in.  Tests affected were

```
FAILED grizli/tests/test_fits.py::FittingTools::test_multibeam - astropy.units.core.UnitConversionError: '' (dimensionless) and 'Angstrom' (length) are not convertible
FAILED grizli/tests/test_fits.py::FittingTools::test_redshift_fit - astropy.units.core.UnitConversionError: '' (dimensionless) and 'Angstrom' (length) are not convertible
FAILED grizli/tests/test_jwst.py::JWSTFittingTools::test_multibeam - astropy.units.core.UnitConversionError: '' (dimensionless) and 'Angstrom' (length) are not convertible
```

Regardless, the code is correct now given the required dependencies.  So this is probably fine for final review.

--- a/grizli/pipeline/auto_script.py
+++ b/grizli/pipeline/auto_script.py
@@ -3386,12 +3386,6 @@ def fine_alignment(field_root='j142724+334246', HOME_PATH='/Volumes/Pegasus/Griz
     """
     Try fine alignment from visit-based SourceExtractor catalogs
     
-    Parameters
-    ----------
-    
-    Returns
-    -------
-    
     """
     import os
     import glob

--- a/grizli/tests/test_jwst.py
+++ b/grizli/tests/test_jwst.py
@@ -182,9 +182,8 @@ class JWSTFittingTools(unittest.TestCase):
             files = glob.glob(f'{conf_path}/*')
             print('Files: ', '\n'.join(files))
 
-        assert(os.path.exists(os.path.join(conf_path,
-                              'GR150C.F115W.conf')))
-        return True
+        assert os.path.exists(os.path.join(conf_path,
+                              'GR150C.F115W.conf'))
     
     def test_multibeam(self):
         """

--- a/grizli/utils.py
+++ b/grizli/utils.py
@@ -6412,7 +6412,7 @@ def fetch_config_files(get_acs=False, get_sky=True, get_stars=True, get_epsf=Tru
         print('Templates directory: {0}/templates'.format(GRIZLI_PATH))
         os.chdir('{0}/templates'.format(GRIZLI_PATH))
 
-        url = 'http://www.stsci.edu/~brammer/Grizli/Files/'
+        url = 'https://www.stsci.edu/~brammer/Grizli/Files/'
         files = [url+'stars_pickles.npy', url+'stars_bpgs.npy']
 
         for url in files:

--- a/setup.cfg
+++ b/setup.cfg
@@ -50,7 +50,6 @@ test =
 docs =
     sphinx
     sphinx-astropy
-    jwst
 jwst =
     jwst
 hst =

